01/global.css

```diff
@@ -1 +1,31 @@
-/*font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;*/
\ No newline at end of file
+/*font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;*/
+h1{
+    margin: 18px;
+    border: 10px;
+    font-size: 300px;
+    font: inherit;
+    vertical-align: baseline;
+    text-align: -webkit-center;
```

Comment:
Nice work so far! You're using the [vendor prefix](https://developer.mozilla.org/en-US/docs/Glossary/Vendor_Prefix) for Webkit (which includes Chrome and SafarI) here, which means that other browsers (Firefox, IE) won't get the `text-align` property value set to `center`. It's better to just use `text-align: center` to have it affect all browsers.

But vendor prefixes are useful when a feature is _experimental_ in CSS. You can read more about them if you'd like.